misc/kind/cluster.yaml

```diff
@@ -22,7 +22,7 @@ kubeadmConfigPatches:
         "service-node-port-range": "32000-32063"
 nodes:
   - role: control-plane
-    image: kindest/node:v1.29.7
+    image: kindest/node:v1.31.0
```

Comment:
I can revert this, but something on my laptop (almost certainly my kubectl version) prevented this from working.